anybox/recipe/odoo/base.py

```diff
@@ -256,7 +256,7 @@ def parse_version(self):
 
         if len(version_split) == 1:
             # version can be a simple version name, such as 6.1-1
-            major_wanted = self.version_wanted[:3]
+            major_wanted = self.version_wanted[:4]
```

Comment:
I guess we have to do something a bit clever to manage version 8, 9 and 10 properly!

```
>>> test = "9.0-1"
>>> test[:3]
'9.0'
>>> test[:4]
'9.1-'
```